### DISCUSSION TITLE
NetBox 4.0.6 error occurs when installing the plugin ver.4.0.0

### DISCUSSION THREAD

**Discussion by 67cosmonaft67** [mattieserver/netbox-topology-views#520] [Q&A] [answered]
Hi,

I upgraded netbox to version 4.0.6 --- it works

After I try installing netbox_topology_views (4.0.0)

source /opt/netbox/venv/bin/activate --- ok

pip3 install netbox-topology-views --- ok

Requirement already satisfied: netbox-topology-views in /opt/netbox/venv/lib/python3.10/site-packages (4.0.0)

insert "netbox-topology-views" in /opt/netbox/local_requirements.txt ---ok

 /opt/netbox/netbox/netbox/configuration.py PLUGINS = ["netbox_topology_views"] ---ok

python3 manage.py migrate netbox_topology_views --- ERROR
(venv) root@netbox-01:/opt/netbox/netbox# python3 manage.py migrate netbox_topology_views
CommandError: No installed app with label 'netbox_topology_views'.

**Comment by mattieserver** [comment, marked as answer]
That error message leads me to believe that something might be wrong in the config.
Please check `/opt/netbox/netbox/netbox/configuration.py` to see if you don't specify the PLUGINS key twice or check for other issues.

**Comment by 67cosmonaft67** [comment]
Hello. Have a good day!
You are right. I made a error and specify the PLUGINS key twice in the configuration file.
Now everything works.
Thanks for the help.
